mirrord/sip/src/lib.rs

```diff
@@ -104,6 +104,9 @@ mod main {
     pub static MIRRORD_TEMP_BIN_DIR_STRING: Lazy<String> =
         Lazy::new(|| get_temp_bin_str_prefix(&MIRRORD_TEMP_BIN_DIR_PATH_BUF));
 
+    /// A default list of processes to be skipped for SIP patching.
+    pub static MIRRORD_SIP_DEFAULT_SKIP_LIST: Lazy<&[&str]> = Lazy::new(|| &["git"]);
```

Comment:
Any reason for this not to be a `const` `[&str; 1]`?

```suggestion
    pub const MIRRORD_SIP_DEFAULT_SKIP_LIST: [&str; 1] = ["git"];
```